frontend/src/assets/css/variables.scss

```diff
@@ -25,7 +25,7 @@ $border-radius-sm: 0.25rem;
 // variable files
 $small-font-size: 14px !default; // Same as fontSizeSmall in typography.ts
 $sub-sup-font-size: 12px !default; // Same as fontSizeTwoSmall in typography.ts
-$code-font-size: $small-font-size !default;
+$code-font-size: 0.875em !default;
 $kbd-font-size: $code-font-size !default;
```

Comment:
Hi @scientia-ac-labore, thanks for your patience as we worked on this internally! 😃 
If you wouldn't mind, could you adjust these two lines as follows:
* `$code-font-size` to be `0.75em` instead of `0.875em`
* `$kbd-font-size:` to be `$small-font-size` - to isolate the code-font-size change from impacting kbd elements

In association with the above changes, we will be adjusting the line-height of our headings to address the overlap issue. I can handle this part and updating cypress tests that will fail as a result. Please feel free to reach out if you have any questions. Thanks again! 🎈

Reply:
I did it. So I am relying on you in next changes and tests fixing :+1: